tests/suites/0_stateless/02_0014_function_maths.sql

```diff
@@ -0,0 +1,4 @@
+SELECT pi();
+SELECT abs(-1);
+SELECT abs(-10086);
+SELECT abs('str');
```

Comment:
SELECT abs('str'); -- {ErrorCode 7}
This will check the error code by the stateless and can't dump the error result.

We shouldn't add the error message with the line no to the result:
```
ERROR 1105 (HY000) at line 4: Code: 7, displayText = Expected numeric types, but got String.
```

Reply:
fixed, thank you